src/ProjectName.h

```diff
@@ -0,0 +1,23 @@
+//
+// Created by 陳世昂 on 2024/10/29.
+//
+
+#ifndef SYLINTERI_PROJECTNAME_H
+#define SYLINTERI_PROJECTNAME_H
+
+#include <iostream>
+
+namespace PROJECT
+{
+    auto print_project_name() -> void {
+        printf("   ______              __     _               _                       _\n");
+        printf(" .' ____ \\            [  |   (_)             / |_                    (_)\n");
+        printf(" | (___ \\_|   _   __   | |   __    _ .--.   `| |-'  .---.   _ .--.   __\n");
+        printf("  _.____`.   [ \\ [  ]  | |  [  |  [ `.-. |   | |   / /__\\ [ `/'`\\]  [  |\n");
+        printf(" | \\____) |   \\ '/ /   | |   | |   | | | |   | |,  | \\__.,  | |      | |\n");
+        printf("  \\______.' [\\_:  /   [___] [___] [___||__]  \\__/   '.__.' [___]    [___]\n");
+        printf("             \\__.'\n");
+    }
```

Comment:
Yo man this is crazy.

Reply:
yep, I know.
btw, I got the idea by the project https://github.com/ImCoriander/ZeroEye .
![](https://private-user-images.githubusercontent.com/165987738/380487887-4ae352c5-5664-4654-be1c-ad005e823f71.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyMjI4MTIsIm5iZiI6MTczMDIyMjUxMiwicGF0aCI6Ii8xNjU5ODc3MzgvMzgwNDg3ODg3LTRhZTM1MmM1LTU2NjQtNDY1NC1iZTFjLWFkMDA1ZTgyM2Y3MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyOVQxNzIxNTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02N2I4YzkzZWQ0ODVmYzYxY2QwNzA0NGI1YWIxZjJlZDFjMmVmZDZmODA4YzU3MDRjN2Q2ZDNlOTRiZGJlYjJjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.c8bkerEV-8l1wqjsAmrhWJxQnrsST3Y7keGPZQu9TI4)

Reply:
I think we can use this instead of hardcoded it into code. 
https://github.com/codewithnick/ascii-art

Reply:
emm....

But it's without modification if use hardcode. and I saw many repo write it like this.

Reply:
I mean the project name will not be modified in the future

Reply:
so hardcode in here is fine, right?

Reply:
ok. have a nice day.